### ISSUE TITLE
Linking entity's to other entity's then the starting one.

### ISSUE THREAD

**Issue by NordicBlueNL** [allfro/canari#13]
Hello,

I am wondering, is it possible to output multiple levels of links towards Maltego?
Following example:
I run a search transform on a custom-made Company entity.
The transform looks in multiple mysql databases for information.
One of the databases is named: KVK

I want the Transform to create a new KVK entity when it finds a result there, and then I want the underlying results from that database linked to the KVK entity, not to the Company entity.

So you would get the following links:
Company -> KVK -> Adress, Name, Phone number etc.

So in short: How do I tell my transform to hook new entitiy's to another entity then the one where the transform runs on?

Thanks in advance for answering!

![Screenshot](https://f.cloud.github.com/assets/721796/317261/06749506-9863-11e2-8dab-eb9332d3fcf5.png)
Just for clarification, I want to run a transform on Bedrijf and have the transform create the KVK phrase and the other 2 entities and have them linked as shown above.
